.mailmap

```diff
@@ -72,6 +73,8 @@ Chew Choon Keat <choonkeat@gmail.com>
 Chris Andrews <cpandrews8@gmail.com> cpandrews8 <cpandrews8@gmail.com>
 Chris Johnson <chris.s.johnson4@gmail.com>
 Chris Young <chris.young@tinder.com> <chris.young@gotinder.com>
+Christian Clauss <cclauss@me.com>
```

Comment:
Is this entry needed, given that we have the following line?

Reply:
@ChALkeR I think it works for `AUTHORS` because the commits happen to be in the right order, and we only add each possibly-already-replaced email once.

This still makes a difference for git itself, though, because it looks up for every commit individually; with this entry:

```
$ git show --use-mailmap bb564a36882102ba8cd46878db7ea886334cac20 | head -n2
commit bb564a36882102ba8cd46878db7ea886334cac20
Author:     Christian Clauss <cclauss@me.com>
```

Without this entry:

```
$ git show --use-mailmap bb564a36882102ba8cd46878db7ea886334cac20 | head -n2
commit bb564a36882102ba8cd46878db7ea886334cac20
Author:     cclauss <cclauss@me.com>
```